### PR TITLE
`ultralytics 8.0.142` add OpenVINO docs

### PR DESCRIPTION
**Pull request by glenn-jocher**
@AyushExel OpenVINO docs page PR 

### Summary
📄🚀🛠️

This pull request enhances the documentation site for Ultralytics by adding a banner, a navigation menu, a reference entry, a redirect, and two new pages for integrations. The new pages cover how to use `tuner` with Ray Tune and how to export a YOLOv8 model to OpenVINO format.

> _Sing, O Muse, of the glorious deeds of Ultralytics_
> _The skillful makers of YOLOv8, the swift and accurate detector_
> _Who in their wisdom and generosity revealed to mortals_
> _The secrets of their integrations, the manifold ways to enhance their models_

### Walkthrough
*  Add new documentation pages for integrations with OpenVINO, Ray Tune, and other tools and platforms ([link](https://github.com/ultralytics/ultralytics/pull/3624/files?diff=unified&w=0#diff-e7feccf2baeb30f2b729c0110dec22a3cc0c1abb1eee8a66b5ecb26e8c2cc382R1-R61), [link](https://github.com/ultralytics/ultralytics/pull/3624/files?diff=unified&w=0#diff-253c092e65b13ab8abf8b5173ab0227ada336ede228f582b64bde2e63f7cfea6L1-R218), [link](https://github.com/ultralytics/ultralytics/pull/3624/files?diff=unified&w=0#diff-0b2e0450d00f6ec28d3259d48f0bd449142341bafb9efe2e754c339e742eea8e))
*  Update `mkdocs.yml` to enable announcement dismissal, create navigation menu for integrations, include `tuner` module in reference, and redirect old hyperparameter tuning page ([link](https://github.com/ultralytics/ultralytics/pull/3624/files?diff=unified&w=0#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2R38), [link](https://github.com/ultralytics/ultralytics/pull/3624/files?diff=unified&w=0#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2R210-R213), [link](https://github.com/ultralytics/ultralytics/pull/3624/files?diff=unified&w=0#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2R359), [link](https://github.com/ultralytics/ultralytics/pull/3624/files?diff=unified&w=0#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2R416))

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
Enhanced Model Export and Workflow Integration in Ultralytics

### 📊 Key Changes
- Fixed syntax for benchmark commands by removing extraneous commas.
- Added exception handling for Dockerfile (GPU) error on TF.js export.
- Updated README files with information about Ultralytics integrations with AI platforms for improved workflows.
- Created new documentation pages explaining Ultralytics integrations and detailing OpenVINO deployment.
- Updated version to '8.0.142' in `__init__.py`.
- Modified the benchmark method in `model.py` to improve the functionality.

### 🎯 Purpose & Impact
- 🛠️ Improved benchmark command syntax ensures accurate model evaluation, resulting in more reliable performance metrics.
- 💻 Enhanced integration information guides users to leverage collaborations like Roboflow, ClearML, Comet, etc., for better model management and deployment.
- 🧠 New integrations documentation, especially for OpenVINO, helps users deploy models more efficiently with performance acceleration.
- ⚙️ Version bump and updates to codebase reflect continuous improvement and preparation for new features and fixes.
- 🌐 The new features and fixes made will contribute to a smoother user experience and enhance the capabilities of the Ultralytics platform for both developers and end-users.

### CODE REVIEW

**Comment by glenn-jocher** [comment]
openvino image

**Comment by AyushExel** [comment]
@glenn-jocher remember to add:
"
The results might vary from device to device and based on the workload of the system
"

**Comment by glenn-jocher** [comment]
> @glenn-jocher remember to add: " The results might vary from device to device and based on the workload of the system "

All done, I put a notice in the docs page.

**Comment by glenn-jocher** [comment]
Adding plotting code here in case we need it in the future:

```python
import matplotlib.pyplot as plt
import pandas as pd

data_fp32 = {
    'YOLOv8n': {
        'PyTorch': 21.79,
        'TorchScript': 23.24,
        'ONNX': 37.22,
        'OpenVINO': 3.29,
    },
    'YOLOv8s': {
        'PyTorch': 31.89,
        'TorchScript': 32.71,
        'ONNX': 43.42,
        'OpenVINO': 3.92,
    },
    'YOLOv8m': {
        'PyTorch': 50.75,
        'TorchScript': 47.90,
        'ONNX': 63.16,
        'OpenVINO': 7.11,
    },
    'YOLOv8l': {
        'PyTorch': 77.45,
        'TorchScript': 85.71,
        'ONNX': 88.94,
        'OpenVINO': 9.37,
    },
    'YOLOv8x': {
        'PyTorch': 100.09,
        'TorchScript': 114.64,
        'ONNX': 110.32,
        'OpenVINO': 15.02,
    },
}

data_fp16 = {
    'YOLOv8n': {
        'PyTorch': 24.39,
        'TorchScript': 23.76,
        'ONNX': 37.37,
        'OpenVINO': 3.29,
    },
    'YOLOv8s': {
        'PyTorch': 30.60,
        'TorchScript': 34.40,
        'ONNX': 43.53,
        'OpenVINO': 3.91,
    },
    'YOLOv8m': {
        'PyTorch': 50.75,
        'TorchScript': 47.90,
        'ONNX': 63.16,
        'OpenVINO': 7.11,
    },
    'YOLOv8l': {
        'PyTorch': 78.55,
        'TorchScript': 84.75,
        'ONNX': 87.71,
        'OpenVINO': 9.38,
    },
    'YOLOv8x': {
        'PyTorch': 94.85,
        'TorchScript': 113.11,
        'ONNX': 111.00,
        'OpenVINO': 15.02,
    },
}

data_xeon = {
    'YOLOv8n': {
        'PyTorch': 24.36,
        'TorchScript': 23.93,
        'ONNX': 39.86,
        'OpenVINO': 11.34,
    },
    'YOLOv8s': {
        'PyTorch': 33.77,
        'TorchScript': 34.84,
        'ONNX': 43.23,
        'OpenVINO': 13.86,
    },
    'YOLOv8m': {
        'PyTorch': 53.91,
        'TorchScript': 53.51,
        'ONNX': 64.16,
        'OpenVINO': 28.79,
    },
    'YOLOv8l': {
        'PyTorch': 75.78,
        'TorchScript': 79.13,
        'ONNX': 88.45,
        'OpenVINO': 56.23,
    },
    'YOLOv8x': {
        'PyTorch': 96.60,
        'TorchScript': 114.28,
        'ONNX': 111.02,
        'OpenVINO': 83.28,
    },
}

data_arc = {
    'YOLOv8n': {
        'PyTorch': 88.79,
        'TorchScript': 102.66,
        'ONNX': 57.98,
        'OpenVINO': 8.52,
    },
    'YOLOv8s': {
        'PyTorch': 189.83,
        'TorchScript': 227.58,
        'ONNX': 142.03,
        'OpenVINO': 9.19,
    },
    'YOLOv8m': {
        'PyTorch': 411.64,
        'TorchScript': 517.12,
        'ONNX': 298.68,
        'OpenVINO': 12.55,
    },
    'YOLOv8l': {
        'PyTorch': 725.73,
        'TorchScript': 892.83,
        'ONNX': 576.11,
        'OpenVINO': 17.62,
    },
    'YOLOv8x': {
        'PyTorch': 988.92,
        'TorchScript': 1186.42,
        'ONNX': 768.90,
        'OpenVINO': 19.00,
    },
}

for data, title in (data_fp16, 'Flex GPU FP16 Precision'), \
        (data_fp32, 'Flex GPU FP32 Precision'),\
        (data_arc, 'Arc GPU'), \
        (data_xeon, 'Xeon CPU'):
    # Grouped bar plot
    ax = pd.DataFrame(data).T.plot(kind='bar', figsize=(10, 6), colormap='viridis')
    ax.set_xlabel('Model Size')
    ax.set_ylabel('Inference Time (ms/im)')
    ax.set_title(f'Inference Time per Image by Model Size and Format ({title})')
    ax.grid(alpha=0.2)
    plt.xticks(rotation=0)
    #plt.legend(title='Format')
    plt.savefig(fname=f'OpenVINO - {title}.jpg', dpi=300, bbox_inches='tight')

```

File: docs/integrations/openvino.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+1. **XML file**: Describes the network topology.
+2. **BIN file**: Contains the weights and biases binary data.
+3. **Mapping file**: Holds mapping of original model output tensors to OpenVINO tensor names.
+
+You can use these files to run inference with the OpenVINO Inference Engine.
+
+## Using OpenVINO Export in Deployment
+
+Once you have the OpenVINO files, you can use the OpenVINO Inference Engine to run the model. The Inference Engine provides a unified API to inference across all supported Intel hardware. It also provides advanced capabilities like load balancing across Intel hardware and asynchronous execution. For more information on using the Inference Engine, refer to the [Inference with OpenVINO Runtime Guide](https://docs.openvino.ai/2023.0/openvino_docs_OV_UG_OV_Runtime_User_Guide.html).
```
Comment (adrianboguszewski) [inline docs/integrations/openvino.md:67]:
```suggestion
Once you have the OpenVINO files, you can use the OpenVINO Runtime to run the model. The Runtime provides a unified API to inference across all supported Intel hardware. It also provides advanced capabilities like load balancing across Intel hardware and asynchronous execution. For more information on running the inference, refer to the [Inference with OpenVINO Runtime Guide](https://docs.openvino.ai/2023.0/openvino_docs_OV_UG_OV_Runtime_User_Guide.html).
```

File: docs/integrations/openvino.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+Remember, you'll need the XML and BIN files as well as any application-specific settings like input size, scale factor for normalization, etc., to correctly setup and use the model with the Inference Engine.
+
+In your deployment application, you would typically do the following steps:
+
+1. Load the model using the `InferenceEngine::Core::ReadNetwork` method.
+2. Prepare the input and output blobs.
+3. Run inference using `InferenceEngine::ExecutableNetwork::Infer`.
```
Comment (adrianboguszewski) [inline docs/integrations/openvino.md:75]:
```suggestion
1. Initialize OpenVINO by creating `core = Core()`.
2. Load the model using the `core.read_model()` method.
3. Compile the model using the `core.compile_model()` function.
4. Prepare the input (image, text, audio, etc.).
5. Run inference using `compiled_model(input_data)`.
```

**Comment by glenn-jocher** [comment]
@ultralytics/intel-openvino ok this is all good to go on our side! Please review and let us know any change you'd like to see, thank you.

File: docs/integrations/openvino.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+For more detailed steps and code snippets, refer to the [OpenVINO documentation](https://docs.openvino.ai/).
```
Comment (adrianboguszewski) [inline docs/integrations/openvino.md:77]:
```suggestion
For more detailed steps and code snippets, refer to the [OpenVINO documentation](https://docs.openvino.ai/) or [API tutorial](https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/002-openvino-api/002-openvino-api.ipynb).
```

File: docs/integrations/openvino.md [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+
+## OpenVINO YOLOv8 Benchmarks
+
+YOLOv8 benchmarks below were run by the Ultralytics team on 4 different model formats measuring speed and accuracy: PyTorch, TorchScript, ONNX and OpenVINO. Benchmarks were run on Intel Flex and Arc GPUs, and on Intel Xeon CPUs.
```
Comment (adrianboguszewski) [inline docs/integrations/openvino.md:81]: The benchmark for Intel Core CPU is missing

File: docs/integrations/openvino.md [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+| yolov8m | ONNX        | ✅      | 99.0      | 0.4999      | 63.16                  |
+| yolov8m | OpenVINO    | ✅      | 49.8      | 0.4997      | 7.11                   |
+| yolov8l | PyTorch     | ✅      | 83.7      | 0.5293      | 78.55                  |
+| yolov8l | TorchScript | ✅      | 167.2     | 0.5268      | 84.75                  |
+| yolov8l | ONNX        | ✅      | 166.8     | 0.5268      | 87.71                  |
+| yolov8l | OpenVINO    | ✅      | 83.8      | 0.5264      | 9.38                   |
+| yolov8x | PyTorch     | ✅      | 130.5     | 0.5404      | 94.85                  |
+| yolov8x | TorchScript | ✅      | 260.7     | 0.5371      | 113.11                 |
+| yolov8x | ONNX        | ✅      | 260.4     | 0.5371      | 111.00                 |
+| yolov8x | OpenVINO    | ✅      | 130.6     | 0.5367      | 15.02                  |
```
Comment (adrianboguszewski) [inline docs/integrations/openvino.md:151]: As the results are almost the same, I'd go here with FP32 only.

File: docs/integrations/openvino.md [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+
+    The benchmarking results below are for reference and might vary based on the exact hardware and software configuration of a system, as well as the current workload of the system at the time the benchmarks are run.
+
+### Flex GPU
+
+The Intel® Data Center GPU Flex Series is a versatile and robust solution designed for the intelligent visual cloud. This GPU supports a wide array of workloads including media streaming, cloud gaming, AI visual inference, and virtual desktop Infrastructure workloads. It stands out for its open architecture and built-in support for the AV1 encode, providing a standards-based software stack for high-performance, cross-architecture applications. The Flex Series GPU is optimized for density and quality, offering high reliability, availability, and scalability.
+
+#### Flex GPU (FP32 Precision)
+
+Benchmark results for Flex GPU at FP32 precision with `half=False`:
```
Comment (adrianboguszewski) [inline docs/integrations/openvino.md:93]: We need to reveal the exact model of Flex, Arc and Xeon e.g. Arc A770, etc.

**Review by adrianboguszewski — COMMENTED** [review]
Thanks, @glenn-jocher. My comments below.

File: docs/integrations/openvino.md (hunk 6 above)
Comment (glenn-jocher) [reply docs/integrations/openvino.md:93]: @AyushExel do you know the Flex, Arc and Xeon models used?

File: docs/integrations/openvino.md (hunk 5 above)
Comment (glenn-jocher) [reply docs/integrations/openvino.md:151]: Ok got it. I'll remove Flex FP16.

Comment (glenn-jocher) [reply docs/integrations/openvino.md]: Done in https://github.com/ultralytics/ultralytics/pull/3624/commits/c694d451a357218ddb8bf2794c6625b34c08f919

File: docs/integrations/openvino.md (hunk 2 above)
Comment (glenn-jocher) [reply docs/integrations/openvino.md:75]: Thanks for the fixes!

File: docs/integrations/openvino.md [hunk 7]
```diff
@@ -0,0 +1,229 @@
+---
+comments: true
+description: 'Export mode: Create a deployment-ready YOLOv8 model by converting it to OpenVINO format. Export to OpenVINO for up to 3x CPU speedup.'
+keywords: ultralytics docs, YOLOv8, export YOLOv8, YOLOv8 model deployment, exporting YOLOv8, OpenVINO, OpenVINO format
+---
+
+<img width="1024" src="https://user-images.githubusercontent.com/26833433/252345644-0cf84257-4b34-404c-b7ce-eb73dfbcaff1.png">
+
+**Export mode** is used for exporting a YOLOv8 model to a format that can be used for deployment. In this guide, we specifically cover exporting to OpenVINO, which can provide up to 3x CPU speedup.
```
[the remark this answers is not in the document]
Comment (glenn-jocher) [reply docs/integrations/openvino.md:9]: @adrianboguszewski should we add some links to iGPU, dGPU, VPU for users to get more info?

File: docs/integrations/openvino.md (hunk 4 above)
Comment (glenn-jocher) [reply docs/integrations/openvino.md]: @adrianboguszewski ah you're right! We are missing Intel Core. Could we get access to a 13th gen Intel Core machine on Intel Devcloud?

@AyushExel FYI

File: docs/integrations/openvino.md [hunk 8]
```diff
@@ -0,0 +1,229 @@
...
+        
+        # Export the model
+        model.export(format='openvino')
+        ```
+    === "CLI"
+    
+        ```bash
+        yolo export model=yolov8n.pt format=openvino  # export official model
+        yolo export model=path/to/best.pt format=openvino  # export custom trained model
+        ```
```
[the remark this answers is not in the document]
Comment (glenn-jocher) [reply docs/integrations/openvino.md:36]: Yes I can add that!

[the remark this answers is not in the document]
Comment (glenn-jocher) [reply docs/integrations/openvino.md]: Updated in https://github.com/ultralytics/ultralytics/pull/3624/commits/f77ec28d0735008b6fe1e25bba5295c6150f2aba

File: docs/integrations/openvino.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (adrianboguszewski) [reply docs/integrations/openvino.md:9]: Sure, let's do it:
CPU: https://docs.openvino.ai/2023.0/openvino_docs_OV_UG_supported_plugins_CPU.html
GPUs: https://docs.openvino.ai/2023.0/openvino_docs_OV_UG_supported_plugins_GPU.html
VPU: https://docs.openvino.ai/2022.3/openvino_docs_OV_UG_supported_plugins_VPU.html

File: docs/integrations/openvino.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (adrianboguszewski) [reply docs/integrations/openvino.md]: Nice!

File: docs/integrations/openvino.md (hunk 6 above)
Comment (adrianboguszewski) [reply docs/integrations/openvino.md]: @AnishaUdayakumar could help with this?

File: docs/integrations/openvino.md (hunk 4 above)
Comment (adrianboguszewski) [reply docs/integrations/openvino.md]: @AnishaUdayakumar, please check if Core is available there

File: docs/integrations/openvino.md (hunk 6 above)
Comment (AnishaUdayakumar) [reply docs/integrations/openvino.md]: @AyushExel Can you confirm the instances used from this list?
https://scheduler.cloud.intel.com/#/systems

File: docs/integrations/openvino.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (glenn-jocher) [reply docs/integrations/openvino.md:9]: Ok perfect, I'll merged your commit and then add the URLs in a new commit.

File: docs/integrations/openvino.md (hunk 6 above)
Comment (adrianboguszewski) [reply docs/integrations/openvino.md]: @glenn-jocher
Intel® Data Center GPU Flex 170
4th Gen Intel® Xeon® Scalable CPU
I don't know your ARC, unfortunately.

Comment (glenn-jocher) [reply docs/integrations/openvino.md]: @AyushExel can you post your Arc model here?

Comment (AyushExel) [reply docs/integrations/openvino.md]: Arc 770 limited edition

Comment (glenn-jocher) [reply docs/integrations/openvino.md]: Resolved in https://github.com/ultralytics/ultralytics/pull/3624/commits/c2fbed4b6993da382a5c090164c5ac0ddb4b3d96

**Comment by glenn-jocher** [comment]
Updated plots in 2:1 aspect ratio.

OpenVINO - Flex GPU FP32 Precision
OpenVINO - Arc GPU
OpenVINO - Xeon CPU
OpenVINO - Core CPU

File: docs/integrations/openvino.md (hunk 4 above)
Comment (glenn-jocher) [reply docs/integrations/openvino.md]: @adrianboguszewski @AnishaUdayakumar @raymondlo84 ok we have new benchmark values by @AyushExel on a 12th Gen Core i9 that I've added in https://github.com/ultralytics/ultralytics/pull/3624/commits/d1e92de175ad8166ef94f4fba68478896233ef4f

If this looks good then we can resolve this final point and all review issues should be resolved!

**Comment by glenn-jocher** [comment]
Core results for 13th Gen Intel Core(TM) i7-13700H

Benchmarks complete for yolov8n.pt on coco128.yaml at imgsz=640 (61.84s)
                   Format Status  Size (MB)  metrics/mAP50-95(B)  Inference time (ms/im)
0                 PyTorch               6.2               0.4478                  104.61
1             TorchScript              12.4               0.4525                  112.39
2                    ONNX              12.2               0.4525                   28.02
3                OpenVINO              12.3               0.4504                   23.53

Benchmarks complete for yolov8s.pt on coco128.yaml at imgsz=640 (93.99s)
                   Format Status  Size (MB)  metrics/mAP50-95(B)  Inference time (ms/im)
0                 PyTorch              21.5               0.5885                  194.83
1             TorchScript              43.0               0.5962                  202.01
2                    ONNX              42.8               0.5962                   65.74
3                OpenVINO              42.9               0.5966                   38.66

Benchmarks complete for yolov8m.pt on coco128.yaml at imgsz=640 (167.62s)
                   Format Status  Size (MB)  metrics/mAP50-95(B)  Inference time (ms/im)
0                 PyTorch              49.7               0.6101                  355.23
1             TorchScript              99.2               0.6120                  424.78
2                    ONNX              99.0               0.6120                  173.39
3                OpenVINO              99.1               0.6091                   69.80

Benchmarks complete for yolov8l.pt on coco128.yaml at imgsz=640 (267.06s)
                   Format Status  Size (MB)  metrics/mAP50-95(B)  Inference time (ms/im)
0                 PyTorch              83.7               0.6591                  593.00
1             TorchScript             167.2               0.6580                  697.54
2                    ONNX             166.8               0.6580                  342.15
3                OpenVINO             167.0               0.0708                  117.69

Benchmarks complete for yolov8x.pt on coco128.yaml at imgsz=640 (358.95s)
                   Format Status  Size (MB)  metrics/mAP50-95(B)  Inference time (ms/im)
0                 PyTorch             130.5               0.6651                  804.65
1             TorchScript             260.8               0.6650                  921.46
2                    ONNX             260.4               0.6650                  526.66
3                OpenVINO             260.6               0.6619                  158.73

File: docs/integrations/openvino.md (hunk 4 above)
Comment (glenn-jocher) [reply docs/integrations/openvino.md]: @adrianboguszewski @AnishaUdayakumar @raymondlo84 good news!! I went out and purchased a 13th Gen Core i7 machine, re-ran all benchmarks and updated the Docs (new table + plot) in https://github.com/ultralytics/ultralytics/pull/3624/commits/b1fb8599f9016ef496c4fd09281066c3214a4cc2.

If this looks good we can resolve this last and final issue.

**Comment by glenn-jocher** [comment]
@adrianboguszewski ok I believe PR is all good to merge now. Once you confirm I will merge, and then the new Docs will be deployed and the page will be live!

**Comment by adrianboguszewski** [comment]
@glenn-jocher, Thanks, please add also to the disclaimer that benchmarking was done with OpenVINO 2023.0. 

@raymondlo84, please look at the PR :)

**Comment by glenn-jocher** [comment]
@adrianboguszewski added note about openvino version 2023.0.1 in https://github.com/ultralytics/ultralytics/pull/3624/commits/0f1b94b36b5c2d7ac10e8b279ff7b057483a477e

**Comment by raymondlo84** [comment]
I'm working with the devcloud team to get the system info on the flex system. I think we have also so mention the spec of the CPU used.

**Comment by raymondlo84** [comment]
Also, is it possible to include the 'script' that was executed somewhere in the footnotes? Just making sure the work is clearly reproducible.

**Comment by glenn-jocher** [comment]
@adrianboguszewski certainly, we can provide the general approach taken to achieve the results in the benchmark.

The PyTorch model was first exported to ONNX using torch.onnx.export(). Then, this ONNX model was converted to OpenVINO using the OpenVINO Model Optimizer. 

During the inference time measurement, for PyTorch, TorchScript and ONNX, the time spent on the model function was measured. For OpenVINO, the time spent on ExecutableNetwork.infer(inputs={input_key: image}) was measured.

As for the hardware specifications, they are detailed within the benchmark documentation. The exact script for benchmarking isn't included due to its complexity and the fact that it's tightly integrated with the rest of our custom benchmarking infrastructure. However, the basic steps mentioned above should help in reproducing our methods.

Please, let us know if you have further queries.

**Comment by glenn-jocher** [comment]
@raymondlo84 I've added a section with steps to reproduce now in https://github.com/ultralytics/ultralytics/pull/3624/commits/97548688ffde4f1a984379bb5dd7a006823bbd9a

Screenshot 2023-07-22 at 15 34 32

**Comment by glenn-jocher** [comment]
@raymondlo84 @adrianboguszewski I've updated the repo description to specifically mention OpenVINO:
Screenshot 2023-07-22 at 15 41 16

I've also added copy to the README linking to the OpenVINO doc page in https://github.com/ultralytics/ultralytics/pull/3624/commits/3567a2fddb1482294bbba724fe454aa3dee8c184.

I'd like to merge with PR as it's been open for two weeks now. I've incorporated all feedback and I believe the main elements are all in place. We can continue to update in the future as required (I update the code and docs daily). Can I go ahead and merge this PR?

**Comment by adrianboguszewski** [comment]
Thanks, @glenn-jocher. Let me confirm this with @raymondlo84 and I'll be back.

**Comment by glenn-jocher** [comment]
@adrianboguszewski thanks! We have our [Intel <> Ultralytics Live Session](https://www.youtube.com/watch?v=81S73zy0wo8&ab_channel=Ultralytics) on Wednesday, so we need the docs up by then to share them with users. Please let me know if there are any final changes required.

**Comment by adrianboguszewski** [comment]
@glenn-jocher, we are ok to merge. When we have some updates, we will open another PR. Sorry, for the long waiting time :)

**Review by adrianboguszewski — APPROVED** [review]

**Comment by glenn-jocher** [comment]
@adrianboguszewski @raymondlo84 @AyushExel OpenVINO PR merged!! Thanks everyone for all your contributions to getting this done!

**Pull request merged**